### DISCUSSION TITLE
Thoughts on exposing this to the internet via Traefik?

### DISCUSSION THREAD

**Discussion by points-unknown** [AbdBarho/stable-diffusion-webui-docker#631] [Q&A]
I tried to expose this container via Traefik and Authelia middleware with the regular traefik labels and assigning port 7860. It loads the webui but throws lots of errors and is not functional

Looking at the errors in the inspection terminal I see:
```
WebSocket connection to 'wss://<<domain name or subdomain>>/queue/join' failed: index.js:488 
```
Any ideas appreciated.

**Comment by DevilaN** [comment]
I use traefik on regular basis. Could you describe your problem in details. Which UI, is it Chrome browser with self signed certificate?
